src/migrations/Install.php

```diff
@@ -123,7 +123,7 @@ protected function createTables()
                     'ownerId'           => $this->integer()->notNull(),
                     'sourceSite'        => $this->integer()->notNull(),
                     'targetSites'       => $this->string(1020)->notNull()->defaultValue(''),
-                    'status'            => $this->enum('status', Constants::ORDER_STATUSES)->defaultValue(Constants::ORDER_STATUS_NEW),
+                    'status'            => $this->enum('status', Constants::ORDER_STATUSES)->defaultValue(Constants::ORDER_STATUS_PENDING),
```

Comment:
@shnsumit, we should also include the `ORDER_STATUS_NEW` in `Install.php`, right?

Reply:
@sidedwards  All new statuses were added to constant so it's going to add from there.

<img width="424" alt="image" src="https://user-images.githubusercontent.com/32937137/147124292-079b0280-1195-4d47-9513-0748a6c62e7d.png">